test/jit/test_misc.py

```diff
@@ -345,3 +345,22 @@ def sum_f(x: BroadcastingList2[float]) -> float:
 
         self.assertTrue(torch.jit.script(sum_i)(4) == 8)
         self.assertTrue(torch.jit.script(sum_f)(4.5) == 9.)
+
+    def test_parse_ir_annotate(self):
+        ir = """
+        graph():
+          %3 : int[] = prim::Constant[value=annotate(List[int], [])]()
+          return (%3)
+        """
+        graph = torch._C.parse_ir(ir, True)
+
+    def test_parse_ir_single_element_tensor(self):
+        ir = """
+        graph():
+          %7 : Long(1, strides=[1], requires_grad=0, device=cpu) = prim::Constant[value={0}]()
+          return (%7)
+        """
+        graph = torch._C.parse_ir(ir, True)
+        func = torch._C._create_function_from_graph("forward", graph)
+        ret = func()
+        self.assertTrue(ret.numel() == 1)
```

Comment:
can you also check that `len(ret.size() == 0)`

Reply:
actually it's ret.size() == 1 for this tensor - added a check